docs/pipelines/remove-pipeline-step.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Remove Step from Pipeline
+intro: Removing unused pipeline steps can help to ensure that only necessary steps are executed, simplifying the management of the pipeline and reducing the risk of errors during the deployment process.
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/applications/add-application
+    next:
+        - /docs/configuration-files/add-configuration-file
```

Comment:
```suggestion
    previous:
    next:
```

I don't think including the removal of pipeline steps should be included in the introduction guide.

Reading about adding an application, then removing a pipeline step, followed by adding a configuration file; doesn't lead to a successful user deployment in the least amount of steps.